src/decisionengine_modules/AWS/transforms/AWSOccupancy.py

```diff
@@ -123,27 +123,25 @@ def fill_capacity_data(self, instances):
   '''
 
 
-@Source.produces(AWS_Occupancy=pd.DataFrame)
-class AWSOccupancy(SourceProxy.SourceProxy):
+@Transform.consumes(spot_occupancy_config=pd.DataFrame)
+@Transform.produces(AWS_Occupancy=pd.DataFrame)
+class AWSOccupancy(Transform.Transform):
     def __init__(self, config):
         super().__init__(config)
+        self.logger = self.logger.bind(
+            class_module=__name__.split(".")[-1],
+        )
```

Comment:
These lines can be removed--the `self.logger` attribute is now provided through the base class and doesn't need to be assigned here.